pages/pricing.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: page
+permalink: /pricing
+title: Pricing
+---
+
+## What is included?
+
+- Ability to deploy applications through Cloud Foundry
+- Self service user management with permissions
+- Access to shared services
+- Buildpacks (Java, Ruby, Node, Go, Python, PHP, Static and Binary) - Custom buildpacks are not supported
+- Security patches
+- Platform updates
+- Documentation
+- ATO tooling (Control Masonry, Shared SSP)
+- Infrastructure management
+- Network security
+
+
+## How much does it cost?
+
+Support tiers:
+- FISMA Low
+  $60k/yr for 3 "systems"
+- FISMA Mod
+  $90k/yr for 1 "system"
```

Comment:
FISMA Moderate requires more "support" in our opinion. Basically a lot more support :)